package/src/components/ImageGallery/ImageGallery.tsx

```diff
@@ -309,6 +307,10 @@ export const ImageGallery = <
    */
   const photoLength = photos.length;
 
+  useEffect(() => {
```

Comment:
What is the reason behind this setImageGalleryAttachments?

Reply:
We need it because we need to filter the current video and manage the state. We are setting this into `imageGalleryAttachments` by this way.